staging/src/k8s.io/apiserver/pkg/endpoints/handlers/trace_util.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package handlers
+
+import (
+	"net/http"
+
+	utiltrace "k8s.io/utils/trace"
+)
+
+func getTraceFields(req *http.Request) []utiltrace.Field {
```

Comment:
s/getT/t/

Reply:
@sttts  Could you give more detail information?

Reply:
oh, just remove the `get` from the function name. It's not a getter.